client/analytics/report/revenue/index.js

```diff
@@ -70,11 +72,20 @@ class RevenueReport extends Component {
 
 			response.json().then( () => {
 				// Ignore data, just use our fake data once we have a response
-				this.setState( { stats: rawData } );
+				this.setState( { stats: mockData } );
 			} );
 		} );
 	}
 
+	onDownload( headers, rows, totals, query ) {
+		return () => {
+			downloadCSVFile(
+				generateCSVFileName( 'revenue', query ),
+				generateCSVDataFromTable( headers, rows, totals )
```

Comment:
@LevinMedia what do you think about including the "totals" row here? In prior csv exports for stats, we only include the raw data - no totals/summary row

Reply:
I think it's a nice touch

Reply:
Only showing the header and rows will work. Thanks!